scripts/linux/trcE2ETest.sh

```diff
@@ -875,7 +880,7 @@ elif [[ "${TEST_NAME,,}" == "${CONNECTIVITY_TEST_NAME,,}" ]]; then
     NETWORK_CONTROLLER_RUNPROFILE=${NETWORK_CONTROLLER_RUNPROFILE:-Offline}
 
     is_build_canceled=$(is_cancel_build_requested $DEVOPS_ACCESS_TOKEN $DEVOPS_BUILDID)
-    if [ $is_build_canceled -eq 1 ]; then
+    if [ "$is_build_canceled" -eq '1' ]; then
```

Comment:
is `"`  synonymous as `'` in bash?

Reply:
No. " interpolates, ' doesn't. So "$is_build_canceled" will be string value of $is_build_canceled and '1' is literal string '1'

Reply:
Ok. I'm just making sure we are not comparing `1`  to `'1'`